### PR TITLE
Fix Stream.mm to record in QT Player, add constants for framerate

### PR DESCRIPTION
**Pull request by iKenndac**
This PR is borne from issue #10, and is inspired by the work done on the same issue over on the Swift port of this repo: https://github.com/seanchas116/SimpleDALPlugin/issues/2

The code changes here use a smaller timing scale than before to allow it to be recorded by QuickTime Player, but always uses a scale that roundly divides into our framerate (which a millisecond scale doesn't necessarily do). It also adds constants for the framerate so changing it later on is much easier if required.

### CODE REVIEW

File: CMIOMinimalSample/Stream.mm [hunk 1]
```diff
@@ -162,19 +165,29 @@ - (CVPixelBufferRef)createPixelBufferWithTestAnimation {
...
+    // like QuickTime Player, we need to be accurate in both our timestamps _and_ have a sensible scale. Using large
+    // timestamps and scales like mach_absolute_time() and NSEC_PER_SEC will work for display, but will error out
+    // when trying to record.
+    //
+    // Instead, we start our presentation times from zero (using the sequence number as a base), and use a scale that's
+    // a multiple of our framerate. This has been observed in parts of AVFoundation and lets us be frame-accurate even
+    // on non-round framerates (i.e., we can use a scale of 2997 for 29,97 fps content if we want to).
+    CMTimeScale scale = FPS * 10;
+    CMTime frameDuration = CMTimeMake(scale / FPS, scale);
+    CMTime pts = CMTimeMake(frameDuration.value * self.sequenceNumber, scale);
```
Comment (johnboiles) [inline CMIOMinimalSample/Stream.mm:185]: Makes sense to me!

**Comment by johnboiles** [comment]
BTW I like your implementation more than the one in seanchas116/SimpleDALPlugin#2 since it avoids adding another instance variable to Stream 🎉

Probably in practice this timestamp is going to come from some other process that sends frames, so it's nice to keep these classes as simple as possible ❤️

**Comment by mizt** [comment]
Fixed a problem with audio input when used with QuickTime.

**Comment by mizt** [comment]
Fixed a problem with audio input when used with QT Player.

**Pull request merged**